src/translations/en.json

```diff
@@ -84,5 +84,9 @@
   "pictureInPicture": {
     "overlay_text": "Playing in Picture In Picture mode",
     "overlay_button": "Play Here"
+  },
+  "smartContainer": {
+    "language": "Language",
```

Comment:
Makes ore sense to put the titles in their respective sections.
There’s no “smart container” component exposed to the app.
Move it to the settings and language sections